docs/faq.md

```diff
@@ -97,19 +98,27 @@ tl;dr Add the following to your `.bazelrc`.
 ```
 # Ensure that sandboxed is added to the spawn strategy list when building with
 # rules_xcodeproj.
-build:rules_xcodeproj --spawn_strategy=sandboxed,remote,worker,local
+build:rules_xcodeproj --spawn_strategy=remote,worker,sandboxed,local
 ```
 
+Alternatively, you can use the [--strategy_regexp] flag to target the relevant targets. For
+instance, if `Sources/BranchSDK/BNCContentDiscoveryManager.m` is not building properly, you can
+specify `--strategy_regexp=Compiling Sources/BranchSDK/.*=sandboxed` to use the `sandboxed` strategy
```

Comment:
This needs quote to handle the space.

Reply:
Sorry, I don't follow. Could you elaborate?

Reply:
There's a space in `Compiling Sources` so I believe it needs to be quoted so that the regex is considered as a single argument instead of two:

```suggestion
specify `--strategy_regexp="Compiling Sources/BranchSDK/.*=sandboxed"` to use the `sandboxed` strategy
```

Reply:
Done. If this looks good, could I get an approval?